docs/wiki/The context - CARML CI environment.md

```diff
@@ -0,0 +1,63 @@
+This page provides an overview of the CARML CI environment. For further details refer to the dedicated Wiki section [The CI environment](./The%20CI%20environment)
+
+---
+
+### _Navigation_
+
+- [Deployment flow](#deployment-flow)
+  - [Module versioning](#module-versioning)
+- [Where does the CARML CI environment fit in?](#where-does-this-platform-fit-in)
+
+---
+
+# Deployment flow
+
+This paragraph provides an overview of the usual deployment flow that goes from source modules to target solutions.
+
+<img src="media\Context\Deployment_flow.png" alt="Deployment flow" height="300">
+
+When it comes to Infrastructure as Code (IaC) deployments, the flow generally covers 3 phases:
+
+1. In the **Develop modules** phase modules are first implemented/updated and then validated using one or multiple test-parameter files, testing their successful deployment against a sandbox subscription to prove their correctness.
+
+1. The next phase, **Publish modules**, publishes the tested and approved modules to a target location for them to be later consumed. The target location should support versioning to allow referencing a specific module version and to avoid breaking changes when referencing them.
```

Comment:
```suggestion
1. The next phase, **Publish modules**, publishes the tested and approved modules to a target location for them to be consumed later. The target location should support versioning to allow referencing a specific module version and to avoid breaking changes when referencing them.
```